### PR TITLE
Abstract away activity/fragment in prompt feature

### PR DESCRIPTION
**Pull request by NotWoods**
---

### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- [ ] **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- [ ] **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

File: components/feature/prompts/src/main/res/values/ids.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <!-- Request code for picking a file from another activity -->
+    <item name="file_picker_activity_request_code" type="id" />
```
Comment (Amejia481) [inline components/feature/prompts/src/main/res/values/ids.xml]: We normally use a `mozac` prefix you can find an example [here](https://github.com/mozilla-mobile/android-components/blob/master/components/browser/toolbar/src/main/res/values/ids.xml)

File: components/feature/prompts/src/test/java/mozilla/components/feature/prompts/PromptContainerTest.kt [hunk 2]
```diff
@@ -0,0 +1,69 @@
+package mozilla.components.feature.prompts
```
Comment (Amejia481) [inline components/feature/prompts/src/test/java/mozilla/components/feature/prompts/PromptContainerTest.kt]: nit: License header

**Comment by NotWoods** [comment]
bors r=Amejia481

**Comment by NotWoods** [comment]
bors retry

**Comment by Amejia481** [comment]
bors retry

**Pull request merged**